GVFS/GVFS/CommandLine/UpgradeVerb.cs

```diff
@@ -30,6 +32,7 @@ public class UpgradeVerb : GVFSVerb.ForNoEnlistment
         {
             this.upgrader = upgrader;
             this.tracer = tracer;
+            this.fileSystem = new PhysicalFileSystem();

```

Comment:
Can we add this line in the default constructor as well. I see a crash with `System.NullReferenceException` without it.

Reply:
Yep nice catch, I am confirming this issue is caught by the full functional tests before pushing a fix.